.github/workflows/release-please.yml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    branches:
+      - main
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      id-token: write
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: node
+          package-name: 'espree'
+          pull-request-title-pattern: 'chore: release${component} ${version}'
```

Comment:
I think that the default here is fine :thinking: 
Its probably the same for the change logs

This is the default:
![image](https://github.com/eslint-community/eslint-plugin-n/assets/2230835/9f0ccceb-659a-40d5-acec-b1c29d0509db)